service_management/test/controllers/analysis/SampleObjectControllerIntegrationSpec.scala

```diff
@@ -188,6 +189,39 @@ class SampleObjectControllerIntegrationSpec
       validateSampleObject(ujs, res.json)
     }
 
+    "delete a specific sample object" in {
+      val all = getAllForTestMuseum
+
+      val expJs    = (all.json \ 2).as[JsObject]
+      val objectId = (expJs \ "objectId").as[String]
+
+      val res =
+        wsUrl(deleteUrl(mid)(objectId)).withHeaders(token.asHeader).get().futureValue
+
+      res.status mustBe OK
+    }
+
+    "try to delete a objectid that not exists, return NotFound" in {
+      val all = getAllForTestMuseum
+
+      val objectId = "123e4567-e89b-12d3-a456-426655440000"
+
+      val res =
+        wsUrl(deleteUrl(mid)(objectId)).withHeaders(token.asHeader).get().futureValue
+
+      res.status mustBe NOT_FOUND
+    }
+
+    "try to delete wrong objectid, return BadRequest" in {
```

Comment:
A clearer test description here would be `"return 400 when trying to delete a sample object with an invalid UUID"`